superset/common/query_context_processor.py

```diff
@@ -447,7 +448,7 @@ def processing_time_offsets(  # pylint: disable=too-many-locals,too-many-stateme
         rv_df = pd.concat(rv_dfs, axis=1, copy=False) if time_offsets else df
         return CachedTimeOffset(df=rv_df, queries=queries, cache_keys=cache_keys)
 
-    def get_data(self, df: pd.DataFrame) -> Union[str, List[Dict[str, Any]]]:
+    def get_data(self, df: pd.DataFrame, coltypes:[]) -> Union[str, List[Dict[str, Any]]]:
```

Comment:
same here

Reply:
to be honest, I'm still wondering if passing along an array is the best way, I'm more used to languages where I would have created an object, or a map, rather than relying on positional informations in arrays.

Maybe a dict would be better. 

To be honest any guidance will be welcome.